packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1642,8 +1646,12 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         _obscureLatestCharIndex = _value.selection.baseOffset;
       }
     }
-
-    if (_isSelectionOnlyChange(value)) {
+    if (value == _value) {
+      // This is possible, for example, when the numeric keyboard is input,
+      // the engine will notify twice for the same value. Perhaps this is an issue.
```

Comment:
Could you create an actual issue for this and link to it here?  I just want to make sure this has the chance to get looked at, in case it is a problem.  It doesn't make sense to me that the engine should notify twice for the numeric keyboard.

Reply:
The track at #65811

Reply:
Thanks!